File: app/src/main/java/com/enzoftware/androidiptv/m3u/ChannelList.java
package com.enzoftware.androidiptv.m3u;

import java.util.ArrayList;

public class ChannelList {
    public String name;
    public ArrayList<ChannelItem> items;
    public ArrayList<String> groups;

    public ChannelList() {
        items = new ArrayList<ChannelItem>();
    }

    public void add(ChannelItem item) {
        items.add(item);
    }
}

File: app/src/main/java/com/enzoftware/androidiptv/m3u/ChannelList.kt
package com.enzoftware.androidiptv.m3u

import java.io.Serializable

class ChannelList : Serializable {
    var name: String? = null
    val items: MutableList<ChannelItem> = mutableListOf()
    var groups: MutableList<String>? = null

    fun add(item: ChannelItem) {
        items.add(item)
    }
}